app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -282,9 +282,11 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
 
     private fun showOtherClassifications(otherClassifications: List<Classifier.Classification>) {
         if (otherClassifications.isEmpty()) {
-            footer.visibility = View.GONE
+            otherPredictionsSeparator.visibility = View.GONE
```

Comment:
The main classification isn't show in this area. Take a look.

![screenshot_20170823-142404](https://user-images.githubusercontent.com/68348/29578404-8ae0c780-8745-11e7-8172-8bf9c2e18ef5.png)